GOFevaluation/utils.py

```diff
@@ -246,25 +360,45 @@ def plot_equiprobable_histogram(data_sample, bin_edges, order=None,
     ax.set_ylim(ylim)
 
     ns = apply_irregular_binning(data_sample, bin_edges, order=order)
+    ns = np.array(ns, dtype=float)
+
+    be = limit_bin_edges(bin_edges, data_sample, order)
+    be_first = be[0]
+    be_second = be[1]
 
     # get colormap and norm for colorbar
     cmap_str = kwargs.get('cmap', 'RdBu_r')
     cmap = mpl.cm.get_cmap(cmap_str)
+
+    if(plot_mode == 'count_density'):
+        ns = density_modifier(ns, be_first, be_second, data_sample)
+        cmap_str = kwargs.get('cmap', 'viridis')
+        cmap = mpl.cm.get_cmap(cmap_str)
+    if(plot_mode == 'num_counts'):
+        cmap_str = kwargs.get('cmap', 'viridis')
+        cmap = mpl.cm.get_cmap(cmap_str)
+
```

Comment:
we should do it explicit:
`if plot_mode == 'sigma_deviation'` ... `elif` ... `elif`...

Reply:
Also then `else: raise ValueError(f'plot_mode {plot_mode} is not defined.')`